src/pages/profiles/ProfileNetworkList.tsx

```diff
@@ -2,25 +2,27 @@ import { FC } from "react";
 import { LxdProfile } from "types/profile";
 import { isNicDevice } from "util/devices";
 import ExpandableList from "components/ExpandableList";
+import ResourceLink from "components/ResourceLink";
 
 interface Props {
   profile: LxdProfile;
+  project: string;
 }
 
-const ProfileNetworkList: FC<Props> = ({ profile }) => {
+const ProfileNetworkList: FC<Props> = ({ profile, project }) => {
   return (
     <>
       {Object.values(profile.devices).some(isNicDevice) ? (
         <ExpandableList
           items={Object.values(profile.devices)
             .filter(isNicDevice)
             .map((device) => (
-              <div
-                key={device.network}
-                className="u-truncate list-item"
-                title={`Network ${device.network}`}
-              >
-                {device.network}
+              <div key={device.network}>
+                <ResourceLink
+                  type="network"
+                  value={device.network}
+                  to={`/ui/project/${project}/network/${device.network}`}
+                />
               </div>
```

Comment:
Can we strip the wrapping div? I think the networks can be displayed inline.

Reply:
I actually way prefer how this looks, I think we utilise the horizontal space efficiently like this. Let's do this across all detail pages?

![Screenshot from 2024-10-17 11-48-29](https://github.com/user-attachments/assets/beaaeeb0-94b0-4d80-9e72-035b14d47b3a)
![image](https://github.com/user-attachments/assets/cb34dbd5-fc84-403e-a4bf-6b8b4a64eb46)

Reply:
Never mind about that, the `project`/`resource` structure does not look good with this inline approach.